static/js/components/search/CountryRefinementOption.js

```diff
@@ -10,17 +10,22 @@ export default class CountryRefinementOption extends React.Component {
     count:    number,
   };
 
-  countryName = (countryCode: string): string => (
-    iso3166.country(countryCode).name
-  );
+  countryName = (countryCode: string): string => {
+    let country, countryName = "-";
+    if (countryCode) {
+      country = iso3166.country(countryCode);
+      countryName = country ? country.name : "-";
+    }
+    return countryName;
```

Comment:
Why do we want these to default to `"-"` here? just wondering.

It might be a little more elegant to wrap the call to `iso3166.country` in `R.defaultTo`, which will return it's first argument if the second one if `null`, `undefined`, or `NaN`. So you could just do:

```js
let country = R.defaultTo("-", iso3166.country(countryCode));
return country === "-" ? country : country.name;
```

something like that!